code/game/machinery/computer/arcade.dm

```diff
@@ -4,24 +4,24 @@
 	icon_state = "arcade1"
 	icon_keyboard = null
 	clicksound = null	//Gets too spammy and makes no sense for arcade to have the console keyboard noise anyway
-	var/list/prizes = list(	/obj/item/weapon/storage/box/snappops					= 2,
-							/obj/item/toy/blink										= 2,
-							/obj/item/clothing/under/syndicate/tacticool			= 2,
-							/obj/item/toy/sword										= 2,
-							/obj/item/weapon/gun/projectile/revolver/capgun			= 2,
-							/obj/item/toy/crossbow									= 2,
-							/obj/item/clothing/suit/syndicatefake					= 2,
-							/obj/item/weapon/storage/fancy/crayons					= 2,
-							/obj/item/toy/spinningtoy								= 2,
-							/obj/random/mech_toy									= 1,
-							/obj/item/weapon/reagent_containers/spray/waterflower	= 1,
-							/obj/random/action_figure								= 1,
-							/obj/random/plushie										= 1,
-							/obj/item/toy/cultsword									= 1,
-							/obj/item/toy/bouquet/fake								= 1,
-							/obj/item/clothing/accessory/badge/sheriff				= 2,
-							/obj/item/clothing/head/cowboy_hat/small				= 2,
-							/obj/item/toy/stickhorse								= 2
+	var/list/prizes = list(	/obj/item/weapon/storage/box/snappops						= 2,

```

Comment:
If you're going to edit every line of this file could you change the tabbing to spaces instead? It displays badly on Github.